src/asm/output.c

```diff
@@ -469,10 +469,10 @@ static void writeFileStackNode(struct FileStackNode const *node, FILE *f)
 	}
 }
 
-static void registerExportedSymbol(struct Symbol *symbol, void *arg)
+static void registerUnregisteredSymbol(struct Symbol *symbol, void *arg)
 {
 	(void)arg;
-	if (sym_IsExported(symbol) && symbol->ID == -1) {
+	if (symbol->src && symbol->ID == -1) {
```

Comment:
I think a comment should be added to explain what the `->src` check is about.

Reply:
A comment that `(void)arg;` is there because `registerUnregisteredSymbol` needs to be a valid callback for `sym_ForEach` would also be helpful.

Reply:
How about: "`// Do not register built-in symbols, which have no symbol->src`"